cmd/ttn-lw-cli/commands/gateways_claim.go

```diff
@@ -38,23 +41,20 @@ var (
 The claiming procedure transfers ownership of gateways using the Device
 Claiming Server.
 
-Authentication of gateway claiming is by the Gateway EUI and the claim
-authentication code (which is part of the gateway entity itself) stored
-in the Identity Server.
+Gateways need to be authorized for claiming before they can be claimeed.
+See: "ttn-lw-cli claim authorize"
 
-Claim authentication code validity is controlled by the owner of the
-gateway by setting the value and optionally a time window when the
-code is valid. As part of the claiming, the claim authentication code
-is not transferred to the new claimed gateway. This prevents further claim
-requests.
+Authentication of gateway claiming is by the Gateway EUI and the claim
+authentication code. For LoRa Basics Station gateways, this code is validated
+in external gateway management servers. This is currently only supported
+for The Things Indoor Gateways.
```

Comment:
```suggestion
authentication code. For LoRa Basics Station gateways, this code is validated
in external gateway management servers. This is currently only supported
for The Things Indoor Gateways.
```

I don't think this belongs in the CLI description